### PR TITLE
Run prettier

### PR DESCRIPTION
**Pull request by cloudhead**

### CODE REVIEW

**Comment by cloudhead** [comment]
Adding prettier as part of our toolchain was such a bad idea..

**Review by CodeSandwich — APPROVED** [review]
Nah, non-negotiable code formatters are a huge relief

File: contracts/Governance/RadicleToken.sol [hunk 1]
```diff
@@ -122,8 +122,8 @@ contract RadicleToken {
         return DECIMALS;
     }
 
-    /* @notice DOMAIN_SEPARATOR */
-    function DOMAIN_SEPARATOR() public view returns (bytes32) {
+    /* @notice domainSeparator */
+    function domainSeparator() public view returns (bytes32) {
```
Comment (cloudhead) [inline contracts/Governance/RadicleToken.sol:126]: I think this is like, a convention / part of the API, so we should keep it in uppercase: https://github.com/ethereum/EIPs/blob/master/assets/eip-712/Example.sol#L35

Comment (CodeSandwich) [reply contracts/Governance/RadicleToken.sol:126]: But on the other hand ERC-20's `decimals` always returns a constant, but is lowercase. Maybe the SCREAMING_SNAKE_CASE convention may not be best suited for the public parts of the API, there are no public fields anyway, just functions :thinking:

Does it need to be public? If it doesn't need to, then we don't need to bend to any conventions for future proofing.

Comment (cloudhead) [reply contracts/Governance/RadicleToken.sol:126]: hmm maybe just drop a message on the radicle token channel

Comment (cloudhead) [reply contracts/Governance/RadicleToken.sol:126]: I'd keep it public just in case, but maybe the case doesn't matter.

Comment (cloudhead) [reply contracts/Governance/RadicleToken.sol:126]: in the EIP: https://eips.ethereum.org/EIPS/eip-712#definition-of-domainseparator it's actually camelCase..

Comment (MrChico) [reply contracts/Governance/RadicleToken.sol:126]: In https://eips.ethereum.org/EIPS/eip-2612#specification it's SCREAMING_SNAKE_CASE

**Pull request merged**